test/onnx/symbolic_opsets/test_symbolic_opset9.py

```diff
@@ -0,0 +1,32 @@
+# Owner(s): ["module: onnx"]
+
+"""Tests for `torch.onnx.symbolic_opset9`."""
+import torch
+from torch import _C
+from torch.onnx import symbolic_opset9 as opset9
+from torch.testing._internal import common_utils
+
+
+class TestPrim(common_utils.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.graph = _C.Graph()
+
+    def test_list_unpack_returns_all_list_elements_when_previous_node_is_list_construct(
+        self,
+    ):
+        # Build the graph
```

Comment:
If there's a way to construct this graph via `torch.jit.script` or `torch.jit.trace`, I think that'd be preferable, since it makes it clear what Python code the test applies to.

Reply:
My concern is that if we constructed the graph via jit, we make assumptions and rely on the implementation of jit and the various onnx passes. Creating a graph this way removes those assumptions.

Reply:
Sure, it's OK to keep this as a unit test, but let's also make sure we have an integration test where we go from Python -> jit Graph -> symbolic function.
Do we have any tests in test_pytorch_onnx_onnxruntime.py that would fail without the change to ListUnpack?
If so, then I'm happy.
If not, could you add a test that starts with Python and would fail without the change to ListUnpack?

Reply:
Good point thanks! I created a test.